### PR TITLE
Jira DOC-897: Style review: replica

### PR DESCRIPTION
**Pull request by lanceleonard**
- Review use of the word "replica" throughout the portfolio to ensure style consistency.

### CODE REVIEW

**Comment by lanceleonard** [comment]
Staged previews for this work [are available](https://docs.redis.com/staging/jira-doc-897/rs/administering/database-operations/replica-ha/).

(Note: the Search tool returns hits from latest, not staged previews.  Use the file list for this PR to see which files have been updated.)

**Review by kaitlynmichael — APPROVED** [review]

File: content/rs/concepts/_index.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ latencies. Resharding is performed without downtime.
 ![Sharding diagram](/images/rs/sharding.png)
 
 *Figure 2*
-*Redis Enterprise places master (M) and slave (S) replicas in separate
+*Redis Enterprise places master node (M) and replica (R) replicas in separate
```
Comment (rrelledge) [inline content/rs/concepts/_index.md:65]:
```suggestion
*Redis Enterprise places master node (M) and replicas (R) in separate
```

File: content/rs/concepts/high-availability/replication.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ guarantees that data is served with minimal interruption.
 You can tune your high availability configuration with:
 
 - [Rack/Zone
-Awareness]({{< relref "/rs/concepts/high-availability/rack-zone-awareness.md" >}}) - When rack-zone awareness is used additional logic ensures that master and slave shards never share the same rack, thus ensuring availability even under loss of an entire rack.
-- [High Availability for Replica Shards]({{< relref "/rs/administering/database-operations/slave-ha.md" >}}) - When high availability
+Awareness]({{< relref "/rs/concepts/high-availability/rack-zone-awareness.md" >}}) - When rack-zone awareness is used additional logic ensures that master and replica shards never share the same rack, thus ensuring availability even under loss of an entire rack.
```
Comment (rrelledge) [inline content/rs/concepts/high-availability/replication.md:20]:
```suggestion
Awareness]({{< relref "/rs/concepts/high-availability/rack-zone-awareness.md" >}}) - When rack-zone awareness is used, additional logic ensures that master and replica shards never share the same rack, thus ensuring availability even under loss of an entire rack.
```

File: content/rs/concepts/shard-placement-policy.md [hunk 3]
```diff
@@ -19,32 +20,32 @@ The shard placement policy helps to maintain optimal performance and resiliency.
...
 
-For example, for a database with two master and two slave shards on a cluster with three nodes and a sparse shard placement policy:
+For example, for a database with two master and two replica shards on a cluster with three nodes and a sparse shard placement policy:
 
 - Node 1 hosts one of the master shards
-- Node 2 hosts the slave for the first master shard
+- Node 2 hosts the replica for the first master shard
 - Node 3 hosts the second master shard
-- Node 1 hosts for the slave shard for master shard 2
+- Node 1 hosts for the replica shard for master shard 2
```
Comment (rrelledge) [inline content/rs/concepts/shard-placement-policy.md:42]:
```suggestion
- Node 1 hosts the replica shard for master shard 2
```

File: content/rs/concepts/terminology.md [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
 alwaysopen: false
 categories: ["RS"]
 ---
-Here are explanations of some of the terms used in Redis Enterprise Software (RS).
+Here are explanations of some of the terms used in Redis Enterprise Software.
 
 ## Node
 
-A node is a physical machine, virtual machine, container or cloud
+A _node_ is a physical machine, virtual machine, container or cloud
```
Comment (rrelledge) [inline content/rs/concepts/terminology.md:13]:
```suggestion
A _node_ is a physical machine, virtual machine, container, or cloud
```

File: content/rs/getting-started/getting-started-docker.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ The [Redis Enterprise Software container](https://hub.docker.com/r/redislabs/red
 represents a node in an RS Cluster. When deploying RS using Docker, there are a couple
 of common topologies:
 
-**Topology #1:** The simplest topology is to run a single-node RS Cluster with a single container in a single host machine. This is best for local development or functional testing. Obviously, single-node clusters come with limited functionality in a few ways. For instance, in a single-node topology, RS can't replicate to slave shards or provide any protection for failures. Simply follow the instruction in the Getting Started pages for Windows, macOS and Linux to build your development environment.
+**Topology #1:** The simplest topology is to run a single-node RS Cluster with a single container in a single host machine. This is best for local development or functional testing. Obviously, single-node clusters come with limited functionality in a few ways. For instance, in a single-node topology, Redis Enterprise Software can't replicate to replica shards or provide any protection for failures. Simply follow the instruction in the Getting Started pages for Windows, macOS and Linux to build your development environment.
```
Comment (rrelledge) [inline content/rs/getting-started/getting-started-docker.md:25]:
```suggestion
**Topology #1:** The simplest topology is to run a single-node RS Cluster with a single container in a single host machine. This is best for local development or functional testing. Obviously, single-node clusters come with limited functionality in a few ways. For instance, in a single-node topology, Redis Enterprise Software can't replicate to replica shards or provide any protection for failures. Simply follow the instruction in the Getting Started pages for Windows, macOS, and Linux to build your development environment.
```

File: content/rs/references/memtier-benchmark.md [hunk 6]
```diff
@@ -7,70 +7,68 @@ alwaysopen: false
 categories: ["RS"]
 aliases: /rs/references/cli-reference/memtier-benchmark/
 ---
-Redis on Flash (RoF) on Redis Enterprise Software (RS) lets you use more cost-effective Flash memory as a RAM extension for your database.
-But what does the performance look like as compared to an all-RAM RS database?
+Redis on Flash (RoF) on Redis Enterprise Software lets you use cost-effective Flash memory as a RAM extension for your database.
 
-The simple scenarios on this page show you how you can get performance results with the free RS trial version.
-If you would like assistance with your evaluation or need to test a larger cluster, we'd be happy to help.
+But what does the performance look like as compared to an memory-only database, one stored solely in RAM?
```
Comment (rrelledge) [inline content/rs/references/memtier-benchmark.md:12]:
```suggestion
But what does the performance look like as compared to a memory-only database, one stored solely in RAM?
```

File: content/rs/references/memtier-benchmark.md [hunk 7]
```diff
@@ -7,70 +7,68 @@ alwaysopen: false
...
 
-For our testing, let's configure a Redis Enterprise Software cluster with the trial version
-and use memtier_benchmark to evaluate the performance of a Redis on Flash enabled database in these scenarios:
 
-- Without replication: 4 master shards
-- With replication: 2 master and 2 slave shards
+## Benchmark and performance test considerations
 
-While it concentrates on RoF, you can also use these steps to evaluate RS without RoF.
+These tests assume you're using a trial version of Redis Enterprise Software and want to test the performance of Redis on Flash enabled database in the following scenarios:
```
Comment (rrelledge) [inline content/rs/references/memtier-benchmark.md:23]:
```suggestion
These tests assume you're using a trial version of Redis Enterprise Software and want to test the performance of a Redis on Flash enabled database in the following scenarios:
```

File: content/rs/references/memtier-benchmark.md [hunk 8]
```diff
@@ -7,70 +7,68 @@ alwaysopen: false
...
+- One non-replicated clustered database with four master shards
+- One highly available and clustered database with two master shards and two replica shards
 
-For the test environment, you must:
+## Test environment and cluster setup
 
-1. Create a three-node RS cluster.
+For the test environment, you need to:
+
+1. Create cluster with three nodes.
```
Comment (rrelledge) [inline content/rs/references/memtier-benchmark.md:39]:
```suggestion
1. Create a cluster with three nodes.
```

File: content/rs/references/memtier-benchmark.md [hunk 9]
```diff
@@ -183,15 +181,17 @@ Where:
...
 
 You should expect to see an average throughput of:
 
-- Around 160,000 ops/sec when testing without replication (i.e. 4 master shards)
-- Around 115,000 ops/sec when testing with enabled replication (i.e. 2 master and 2 slave shards)
+- Around 160,000 ops/sec when testing without replication (i.e. Four master shards)
+- Around 115,000 ops/sec when testing with enabled replication (i.e. Four master and 2 replica shards)
 
-In both cases the average latency should be sub-millisecond.
+In both cases, the average latency should below one millisecond.
```
Comment (rrelledge) [inline content/rs/references/memtier-benchmark.md:197]:
```suggestion
In both cases, the average latency should be below one millisecond.
```

File: content/rs/release-notes/rs-5-6-0-april-2020.md [hunk 10]
```diff
@@ -131,7 +131,7 @@ With build 5.6.0-39:
 ## Known limitations
 
 ### Active-Active databases
-- RS51359 - Active-Active databases, using replication and Append Only File (AOF) for [Database Persistence]({{< relref "rs/concepts/data-access/persistence.md" >}}) are suffering from memory leak on slave the shards causing them to grow bigger than the master shards. Customers are advised to upgrade to RS 6.0.12 ***TBD***. Meanwhile you can use snapshot for Database Persistence or restart the slave shards ***TBD***.
+- RS51359 - Active-Active databases, using replication and Append Only File (AOF) for [Database Persistence]({{< relref "rs/concepts/data-access/persistence.md" >}}) are suffering from memory leak on replicas. The shards causing them to grow bigger than the master shards. Customers are advised to upgrade to RS 6.0.12. Meanwhile, you can use snapshot for Database Persistence or restart the replica shards.
```
Comment (rrelledge) [inline content/rs/release-notes/rs-5-6-0-april-2020.md:134]:
```suggestion
- RS51359 - Active-Active databases, using replication and Append Only File (AOF) for [Database Persistence]({{< relref "rs/concepts/data-access/persistence.md" >}}), are suffering from memory leaks on replica shards, causing them to grow bigger than the master shards. Customers are advised to upgrade to RS 6.0.12. Meanwhile, you can use a snapshot for Database Persistence or restart the replica shards.
```

**Review by rrelledge — APPROVED** [review]
Looks good. I left some suggestions for typo fixes.

Also, here's a PR to fix the glossary mini-toc: https://github.com/RedisLabs/redislabs-docs/pull/1543

File: content/rs/administering/database-operations/replica-ha.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
+---
+Title: High availability for replica shards
+linkTitle: High availability for replicas
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/administering/database-operations/slave-ha.md,
+         /rs/administering/database-operations/slave-ha/,
+         /rs/administering/database-operations/replica-ha.md,
```
Comment (banker) [inline content/rs/administering/database-operations/replica-ha.md:10]: @lanceleonard Why is the page itself mentioned in aliases here?

**Review by banker — APPROVED** [review]
LGTM. Thanks for attending to this so quickly.

File: content/rs/administering/database-operations/replica-ha.md (hunk 11 above)
Comment (lanceleonard) [reply content/rs/administering/database-operations/replica-ha.md:10]: @banker - It's part of a test I'm doing with redirects.  (We've been seeing some strange behavior and we're trying to isolate the root cause.)

**Pull request merged**